packages/xo-web/src/common/editable/index.js

```diff
@@ -388,10 +410,34 @@ export class Select extends Editable {
     return children || <span>{renderer ? renderer(value) : value}</span>
   }
 
+  _selectAll = () => {
+    const { onChange, options } = this.props
+    return onChange(options)
+  }
+
   _renderEdition () {
-    const { saving, valueKey } = this.state
-    const { options } = this.props
+    const { hasSelectAll, multi, options } = this.props
+    if (multi || hasSelectAll) {
+      return (
+        <form
+          className='input-group'
+          ref={form => {
+            form && this.setState({ form })
+          }}
+        >
+          <ReactSelect {...this.props} />
+          <span className='input-group-btn'>
+            <Tooltip content={_('selectAll')}>
+              <Button onClick={this._selectAll} style={ADDON_BUTTON_STYLE}>
+                <Icon icon='add' />
+              </Button>
+            </Tooltip>
+          </span>
```

Comment:
After thinking about it, let's remove the "Select all" button to keep the editable input simple and because it should be natively supported once we upgrade to react-select v2.